code/modules/projectiles/projectile/bullets/sniper.dm

```diff
@@ -1,56 +1,97 @@
-// .50 (Sniper)
+// .50 BMG (Sniper)
 
 /obj/projectile/bullet/p50
-	name =".50 bullet"
+	name =".50 BMG bullet"
 	speed = 0.4
 	range = 400 // Enough to travel from one corner of the Z to the opposite corner and then some.
 	damage = 70
 	paralyze = 100
 	dismemberment = 50
+	catastropic_dismemberment = TRUE
 	armour_penetration = 50
+	///If TRUE, deal additional damage to objects.
 	var/breakthings = TRUE
+	///Determines how much additional damage the round does to mechs. For most rounds, needs breakthings to be TRUE.
+	var/mecha_damage = 30
 
 /obj/projectile/bullet/p50/on_hit(atom/target, blocked = 0)
 	if(isobj(target) && (blocked != 100) && breakthings)
-		var/obj/O = target
-		O.take_damage(80, BRUTE, BULLET, FALSE)
+		var/obj/thing_to_break = target
+		var/damage_to_deal = damage * 2
+		if(ismecha(thing_to_break))
+			damage_to_deal += mecha_damage
+		thing_to_break.take_damage(damage_to_deal, BRUTE, BULLET, FALSE)
 	return ..()
 
-/obj/projectile/bullet/p50/soporific
-	name =".50 soporific bullet"
+/obj/projectile/bullet/p50/surplus
+	name =".50 BMG surplus bullet"
 	armour_penetration = 0
-	damage = 0
+	paralyze = 0
 	dismemberment = 0
+	catastropic_dismemberment = FALSE
+
+/obj/projectile/bullet/p50/disruptor
+	name =".50 BMG disruptor bullet"
+	damage_type = STAMINA
 	paralyze = 0
+	dismemberment = 0
+	catastropic_dismemberment = FALSE
 	breakthings = FALSE
+	mecha_damage = 150 //An exception to the rule, this deals damage to mechs and silicons but not objects.
```

Comment:
mechs are surprisingly weak especially in health, this utterly cores a gygax in two shots and probably still gives it a lot of problems in one, especially with the EMP. just take that in note.

Reply:
Having tested it a bit, I think maybe you're not wrong about it being a little much. I'll bring these values down a bit. Same with standard shot.